_data/simple-icons.json

```diff
@@ -7996,6 +7996,11 @@
             "hex": "3A98D9",
             "source": "https://storify.com"
         },
+        {
+            "title": "Storyblok",
+            "hex": "0AB3AF",
```

Comment:
The hex in the `colored-standalone.svg` is `#09B3AF` - should we change to that?

Reply:
Updated